projects/plugins/jetpack/extensions/blocks/blogging-prompt/style.scss

```diff
@@ -0,0 +1,48 @@
+.jetpack-blogging-prompts {
+	border: 1px solid #dcdcde;
```

Comment:
I think we could use variables in this file, by importing `@import '@automattic/jetpack-base-styles/gutenberg-base-styles';`

See https://github.com/Automattic/jetpack/blob/4175424728e0e168f4155084782d750c4683497f/projects/plugins/jetpack/extensions/README.md#L345-L352 for more info.

Reply:
Nice idea, I will check in with @ollierozdarz who worked on initial designs for the block and handle that in a follow-up.